src/networks.json

```diff
@@ -1566,5 +1566,19 @@
     "explorer": "https://turbulent-unique-scheat.explorer.mainnet.skalenodes.com/",
     "start": 401478,
     "logo": "ipfs://QmUdwAZJfyKGZnfPGDsCnNvGu123mdd57kTGj1Y3EWVuWK"
+  },
+  "43288": {
+    "key": "43288",
+    "name": "Boba Avax L2",
+    "shortName": "Boba-Avax",
+    "chainId": 43288,
+    "network": "mainnet",
+    "multicall": "0x352E11Da7C12EA2440b079A335E67ff9219f6FfB",
```

Comment:
Can verify and publish code here https://blockexplorer.avax.boba.network/address/0x352E11Da7C12EA2440b079A335E67ff9219f6FfB/contracts#address-tabs

Reply:
@ChaituVR  Whoops! sorry about that i could have swore i had done that. It is done now :)

Reply:
Thanks @dirtycajunrice